uberfire-extensions/uberfire-widgets/uberfire-widgets-core/uberfire-widgets-core-editors/src/main/java/org/uberfire/ext/widgets/core/client/resources/i18n/EditorsConstants.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2015 JBoss, by Red Hat, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.uberfire.ext.widgets.core.client.resources.i18n;
+
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.i18n.client.Messages;
+
+/**
+ * Editors Widgets I18N constants
+ */
+public interface EditorsConstants
```

Comment:
I think this should be `enum`

Reply:
This also came from other modules, I just copied to this separated package